src/icons/flowTypes.js

```diff
@@ -1,17 +1,19 @@
 // @flow
+import type { Element } from 'react';
+
 export type Icon = {
     className?: string,
     color?: string,
     height?: number,
-    /** A text-only string describing the icon if it's not purely decorative for accessibility */
-    title?: string | React.Element<any>,
+    /** A string describing the icon if it's not purely decorative for accessibility */
+    title?: string | Element<any>,
```

Comment:
btw I think we can make this all React.Node now.
Though not even sure what title element translates to though, maybe `<title>` is fine having elements in it for accessibility

Reply:
yah, we inherited this type from existing icons, so I think we can lock down the types more, encouraging people to use `intl.formatMessage` or some other method to insert the text.

Checked yesterday, and title is allowed to have elements in : https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title#Usage_notes